includes/Admin/Admin_Page.php

```diff
@@ -63,6 +64,29 @@ public function render_page() {
 
 		$available_plugins = $this->get_available_plugins();
 
+		$selected_plugin_basename = filter_input( INPUT_GET, 'plugin', FILTER_SANITIZE_STRING );
+
 		require_once WP_PLUGIN_CHECK_PLUGIN_DIR_PATH . '/templates/admin-page.php';
 	}
+
+	/**
+	 * Add "check this plugin" link in the plugins list table.
+	 *
+	 * @param array  $actions     List of actions.
+	 * @param string $plugin_file Plugin main file.
+	 * @return array
+	 */
+	public function filter_plugin_action_links( $actions, $plugin_file ) {
+
+		if ( is_plugin_active( $plugin_file ) ) {
```

Comment:
This condition isn't needed, we should also be able to test plugins that aren't active.

The only critical differentiation that we had previously discussed is: Only for _active_ plugins we should run the runtime checks.

Reply:
Got it, thank you! 👍